modules/config_pr_bitbucket/src/RepoControllers/BitBucketController.php

```diff
@@ -3,130 +3,308 @@
 namespace Drupal\config_pr_bitbucket\RepoControllers;
 
 use Drupal\config_pr\RepoControllerInterface;
+use Bitbucket\API\Authentication;
+use Bitbucket\API\Http\Client;
+use Bitbucket\API\Repositories;
+use Bitbucket\API;
 
 /**
  * Class to define the BitBucket controller.
  *
  * @see \Drupal\config_pr\RepoControllerInterface
  */
-class BitBucketController implements RepoControllerInterface {
+class BitBucketController implements RepoControllerInterface
+{
 
   /**
    * Holds the controller name.
    *
-   * @var string $name.
+   * @var string $name .
    */
   protected $name = 'BitBucket';
 
   /**
    * Holds the controller Id.
    *
-   * @var string $id.
+   * @var string $id .
    */
   protected $id = 'config_pr_bitbucket.repo_controller.bitbucket';
 
+  /**
+   * @var $repo_user
+   *   The repo user
+   */
+  private $repo_user;
```

Comment:
You don't need to do this, these variables are defined on the class being extended